packages/migrate/src/migrate.ts

```diff
@@ -136,7 +136,7 @@ export async function* migrate(input: MigrateInput): AsyncGenerator<string> {
     .queue(
       LintPlugin,
       {
-        eslintOptions: { cwd: basePath, useEslintrc: true, fix: true },
+        eslintOptions: { cwd: packageDir, useEslintrc: true, fix: true },
```

Comment:
`packageDir` here is the targetPath derived packageDir?

When these plugins run, why is it important to pass the dir to these plugins?

Reply:
Correct. Eslint is looking for its config starting from cwd value